code/style.css

```diff
@@ -1,7 +1,657 @@
+  width: 100%;
+  height: 30px;
+  border: none;
+  background-color: #f5f5f5;
+  font-size: 14px;
+  font-family: 'Poppins', Sans-Serif;
+  color: #696969;
+}
+
+.form-select-box:focus, .form-input-date:focus, .form-input-text:focus {
+  outline: none;
+  background-color: white;
+  border: solid 1px #999 
+}
+
+.form-select-box:hover, .form-input-date:hover, .form-input-text:hover  {
+  background-color: white;
+  border: solid 1px #999 
+}
+
+  /* some validation styling */
+.form-select-box:valid, .form-input-date:valid {
+  background-color: #dff4e9;
+}
+
+button.form {
+  width: 100px;
+  background-color: #033150;
+  border: none;
+  color: white;
+  padding: 10px 5px;
+  text-decoration: none;
+  font-family: 'Poppins', Sans-Serif;
+  cursor: pointer;
+  margin-top: 20px;
+  outline: none;
+  border-radius: 4px;
+}
+
+button.form:hover {
+  background-color: #043d64;
+}
+
+/* For tablet only */
+@media screen and (min-width: 667px) and (max-width: 1024px) {
+
+  .form-size-group {
+    width: 25%;
+  }
+}
+
+/* For computer only */
+@media (min-width: 1025px) {
+
+  section.form {
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 100px;
+  }
+
+  .form-textbox {
+    width: 23%;
+    margin-right: 5%;
+    height: 450px;
+    overflow: auto;
+  }
+
+  .form-textbox-header {
+    font-size: 36px;
+  }
+
+  form {
+    justify-content: space-between;
+    height: 450px;
+    width: 37%;
+    padding: 20px 30px 30px 30px;
+  }
+
+  .form-select-group {
+    width: 45%;
+  }
+  
+  .form-size-group {
+    width: 25%;
+  }
+
+  .form-input-date {
+    width: 100%;
+  }
+
+  .form-dates-group {
+    width: 45%;
+    margin: 0;
+  }
+
+  h3.form-label-heading {
+    margin: 10px 0 0 0;
+  }
+
+  .form-dates-checkbox-group {
+    width: 45%;
+    justify-content: space-between;
+    margin: 0;
+  }
+
+  .form-extra-group {
+    width: 100%;
+  }
+
+  button.form {
+    margin-top: 20px;
+  }
+}
+
+/* FAQ SECTION */
+
+section.accordion {
+  width: 100%;
+  margin-top: 30px;
+  display: flex;
+  justify-content: center;
+}
+
+.accordion-container {
+  background-color: #033150;
+  color: white;
+  width: 80%;
+}
+
+.accordion-heading {
+  font-family: 'Abril Fatface', Cursive;
+  font-size: 30px;
+  font-weight: normal;
+  margin: 0;
+  padding: 20px;
+}
+
+.accordion-element {  
+  width: 100%;
+  border-top: solid 1px white;
+  margin-bottom: 0;
+}
+
+.accordion-header {
+  background-color: #033150;  
+  color: white;
+  font-family: 'Poppins', Sans-Serif;
+  font-size: 16px;
+  font-weight: lighter;
+  text-transform: uppercase;
+  text-align: left;
+  outline: none;
+  border: none;
+  padding: 20px;
+  margin: 0;
+  width: 100%;
+}
+
+/* changing the mouse cursor on hover */
+.accordion-header:hover {
+  cursor: pointer;
+}
+
+/* using css to make every other row in the accordion have another background color */
+#accordion-header-1, #accordion-header-3 {
+  background-color: #043d64;
+}
```

Comment:
There's a really useful CSS selector you could use for creating a different styling for every other element. Take a look at n:th child here: https://www.w3schools.com/cssref/sel_nth-child.asp